app/validators/preprint-form-validator.js

```diff
@@ -0,0 +1,38 @@
+import { validator, buildValidations } from 'ember-cp-validations';
+
+const Validations = buildValidations({
+    title: {
+        description: 'Title',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                // minimum length for title?
+                max: 300,
```

Comment:
Out of curiosity how did you arrive at this number (and why)?

For comparison, I looked in the OSF source code to see what the underlying field allows. There, the requirements are `1<=title<=200` characters:

https://github.com/abought/osf.io/blob/dfd3d4e332fcc5534fa385515f288b6848b9a5e6/website/project/model.py#L731-L731